openstack/neutron/templates/deployment-cc-fabric-agent.yaml

```diff
@@ -30,6 +30,9 @@ spec:
         pod.beta.kubernetes.io/hostname:  cc-fabric-{{ $platform }}-agent
         configmap-etc-hash: {{ include (print $.Template.BasePath "/configmap-etc.yaml") $ | sha256sum }}
         configmap-etc-cc-fabric: {{ include (print $.Template.BasePath "/configmap-etc-cc-fabric.yaml") $ | sha256sum }}
+        prometheus.io/scrape: "true"
+        prometheus.io/targets: "{{ required "$.Values.alerts.prometheus missing" $.Values.alerts.prometheus }}"
+        prometheus.io/targets: "9090"
```

Comment:
Oops, yes, you're right, I accidentally duplicated `target`. Though in the meantime @databus23 commented on this as well and said ports are autodiscovered and we shouldn't add them to avoid double-scraping.